db-scheduler/src/test/java/com/github/kagkarlsson/scheduler/logging/LogHelperTest.java

```diff
@@ -0,0 +1,46 @@
+package com.github.kagkarlsson.scheduler.logging;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class LogHelperTest {
```

Comment:
I hope a test of this kind is enough. I didn't really want to refactor much of the existing test code...

Reply:
Yeah, that should be fine. No need to verify in larger tests I think.